Source/init.cpp

```diff
@@ -169,8 +171,48 @@ std::vector<std::string> GetMPQSearchPaths()
 	return paths;
 }
 
+bool CheckExtraFontsVersion(AssetRef &&ref)
+{
+	const size_t size = ref.size();
+	AssetHandle handle = OpenAsset(std::move(ref), false);
+	if (!handle.ok())
+		return true;
+
+	std::unique_ptr<char[]> version_contents { new char[size] };
+	if (!handle.read(version_contents.get(), size))
+		return false;

```

Comment:
Is it better to assume the fonts MPQ is out of date if we're unable to read from the VERSION file? We do the same for if the asset handle is not okay.
```suggestion
	std::unique_ptr<char[]> version_contents { new char[size] };
	if (!handle.read(version_contents.get(), size))
		return true;
```